spec/features/regular_user/user_registration_spec.rb

```diff
@@ -17,15 +17,15 @@
     @user = User.last
   end
   it "I can see my user name displayed" do
-    expect(page).to have_content("Hello, #{@user.name}")
+    expect(page).to have_content("#{@user.name}")
```

Comment:
checking these values is much better testing... the html is more of a frontend test responsibility...